src/content/docs/logs/logs-context/get-started-logs-context.mdx

```diff
@@ -161,14 +165,29 @@ For more on these options, see the agent-specific logs-in-context docs.
 
 ## Log metrics [#log-metrics]
 
-The logging metrics reported by <InlinePopover type="apm" /> agents are displayed on the <DoNotTranslate>**Logs**</DoNotTranslate> chart on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page. These metrics are separate from the [log forwarding feature](#apm-logs-defined) and cannot be disabled at the account level. To disable logging metrics, see the dedicated APM configuration docs (for example, [this `logging_metrics` config option for PHP](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#cfg-application_logging_metrics-enabled)).
+When you set up an <InlinePopover type="apm" /> agent, you automatically get a **Logs** chart of logging metrics on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page: 
```

Comment:
Logs chart of logging metrics seems redundant, do we need it in both places?

Reply:
@jeff-colucci good catch! I changed it to this: When you set up an <InlinePopover type="apm" /> agent, you automatically get a chart of logging metrics on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page: